pyproject.toml

```diff
@@ -29,34 +28,34 @@ keywords = [
 
 
 [tool.poetry.dependencies]
-python = "^3.6"
+python = ">=3.7,<4"
 lark-parser = "0.11.0"
 cachetools = ">=4.2.4"
 python-dateutil = ">=2.8.2"
 
 [tool.poetry.dev-dependencies]
```

Comment:
`tool.poetry.group.test.dependencies`

Reply:
Not allowed because of the poetry version allowed for the python 3.7

Reply:
👍